app/models/transaction.rb

```diff
@@ -49,6 +50,9 @@ def total
       where('bank_account_transactions.currency' => currency) }
   scope :incomes,     -> { joins(:category).where('categories.type' => Category::CATEGORY_INCOME)}
   scope :expenses,    -> { joins(:category).where('categories.type' => Category::CATEGORY_EXPENSE)}
+  scope :without_transfers, -> { joins('LEFT OUTER JOIN categories category_transactions
```

Comment:
@yunixon why do you use `left outer join` and check id category is null? As I remember category is persisted.

And instead make join and load whole category table you may only load id of transfer category and filter by id